### PR TITLE
fix: force docker to use linux/amd64 platform

### LINKED ISSUE #767: Make sure that docker images can run on ARM platforms
Currently we only have amd64 docker images which are not able to run on arm64 devices by default. As an immediate fix we can force docker (or podman) to use `linux/amd64` platform which makes them to use emulation (`Rosetta 2` under mac and `QEMU` under linux).

### PR DESCRIPTION
**Pull request by mabdollahpour-ol**
Using `--platform linux/amd64` makes it possible to run amd64 images on arm64 platforms through emulation

Closes #767

### CODE REVIEW

File: scripts/release_scripts/run_macaron.sh [hunk 1]
```diff
@@ -585,6 +585,7 @@ else
 fi
 
 docker run \
+    --platform=linux/amd64 \
```
Comment (behnazh-w) [inline scripts/release_scripts/run_macaron.sh:588]: Can you please add a comment explaining why this option is added?

Comment (mabdollahpour-ol) [reply scripts/release_scripts/run_macaron.sh:588]: It seems it's not possible to add comment in the middle of a command in bash. I have to put this flag in a separate variable and put the comment on top of that. Is that ok?

Comment (behnazh-w) [reply scripts/release_scripts/run_macaron.sh:588]: You can just add the comment before the `docker run` command. No need to add a variable.

**Review by behnazh-w — APPROVED** [review]

**Review by tromai — APPROVED** [review]
LGTM. Thanks for the change.

**Pull request merged**